viewing_party/party.py

```diff
@@ -0,0 +1,158 @@
+def create_movie(title,genre,rating):
+    create_movie_dictionary = {
+        "title": title,
+        "genre": genre,
+        "rating": rating
+    }
+    if title and genre and rating:
+        return create_movie_dictionary
+    else: 
+        return None
+
+def add_to_watched(user_data,movie):
+    watched_list = user_data["watched"]
+    watched_list.append(movie)
+    return user_data
+
+def add_to_watchlist(user_data,movie):
+    add_watchlist = user_data["watchlist"]
+    add_watchlist.append(movie)
+    return user_data
+
+
+def watch_movie(user_data, title):
+    for movie in user_data['watchlist']:
+        if title == movie["title"]:
+            user_data["watchlist"].remove(movie)
+            user_data["watched"].append(movie)
+    return user_data
+
+
+
+def get_watched_avg_rating(user_data):
+    
+    
+    if len(user_data['watched']) > 0:
+        counter = 0 
+        for movie in user_data["watched"]:
+            rating = movie["rating"]
+            counter += rating 
+        average = counter / len(user_data["watched"])
+        return average
+    else:
+        return 0.0
+
+def get_most_watched_genre(user_data):
+    genre_list = []
+    if len(user_data['watched']) > 0:
+        for movie in user_data["watched"]:
+            for genre in movie["genre"]:
+                genre = movie["genre"]
+                genre_list.append(genre)
+        most_common = max(genre_list, key = genre_list.count)
+        return most_common
+    else:
+        return None
+            
+
+def get_unique_watched(user_data):
```

Comment:
👍 This works, but I suggest you consider using a `set` when you need to get a list of unique items.